modules/statics/src/coins.ts

```diff
@@ -7949,6 +7957,30 @@ export const coins = CoinMap.fromCoins([
     undefined,
     Networks.test.goerli
   ),
+  terc20(
+    '29c91204-eab5-499c-a623-1604c37b3848',
+    'tusdc',
+    'Goerli USDC',
+    18,
```

Comment:
Is this the underlying contract in etherscan? If so then it says the token has 6 decimals: https://goerli.etherscan.io/token/0x07865c6e87b9f70255377e024ace6630c1eaa37f no?

Reply:
Is not the shared address shared in the ticket, i think maybe the ticket address was wrong

https://bitgoinc.atlassian.net/browse/EA-2134

Reply:
Removed to push the prod tokens